src/main/java/internet/shop/util/ConnectionUtil.java

```diff
@@ -0,0 +1,30 @@
+package internet.shop.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    static {
+        try {
+            Class.forName("com.mysql.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Can't find mysql Driver", e);
+        }
+    }
+
+    public static Connection getConnection() {
+        Properties dbProperties = new Properties();
+        dbProperties.put("user", "root");
+        dbProperties.put("password", "p81gORq76");
```

Comment:
never push your real credentials to github

Reply:
yeah ive got it too late :D